ocfweb/account/templates/account/register/index.html

```diff
@@ -118,9 +123,76 @@
                 </p>
             </div>
 
-            <hr />
+            <div class="well well-sm">
+                <strong>Available Account Names</strong>
+                <ul class="list-group list-group-flush" id="recommendations" style="margin: 7px"></ul>
+                <button type="button" class="btn btn-secondary" onclick="recommend()">More</button>
+            </div>
+            <hr class="visible-sm" />
 
             {% include "account/partials/password-requirements-sidebar.html" %}
+
         </div>
     </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
We already link to jquery in [`base.html`](https://github.com/ocf/ocfweb/blob/master/ocfweb/templates/base.html#L227) ([and we also redundantly vendor it in for some reason](https://github.com/ocf/ocfweb/blob/master/ocfweb/static/js/jquery-1.11.3.min.js)). If you need a different version, we should try and see if we can upgrade it site-wide instead of linking it twice.